crest/Assets/Crest/Crest/Shaders/Ocean.shader

```diff
@@ -419,13 +419,8 @@ Shader "Crest/Ocean"
 
 			half4 Frag(const Varyings input, const float facing : VFACE) : SV_Target
 			{
-				float pixelZ = LinearEyeDepth(input.positionCS.z);
 				#if _UNDERWATER_ON
 				const bool underwater = IsUnderwater(facing, _ForceUnderwater);
-				if(underwater)
-				{
-					clip(_CrestVolumeExtinctionLength - pixelZ);
```

Comment:
Although it's an extra branch, won't the fact that this could be happening for a fair few fragments in parallel mean that it could be a potential saving?